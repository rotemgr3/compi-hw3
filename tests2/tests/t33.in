 
//piazza 95 - should be errorFuncNoOverride +
int foo(int a) {int booga = 5;}
override int foo(int a, int y) {int booga = 5;}
void main() {int booga = 5;}
