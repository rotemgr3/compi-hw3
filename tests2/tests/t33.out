---end scope---
a INT -1
booga INT 0
line 4: function foo was declared before as non-override function
